src/extension.ts

```diff
@@ -530,6 +563,7 @@ export async function activate(context: vscode.ExtensionContext) {
                 if (currentPanel) {
                     console.log("Sending clearing state command");
                     currentPanel.webview.postMessage({
+                        active_device: currentActiveDevice,
                         command: "reset-state",
```

Comment:
this is a comment for the future but should we have these commands as constants somewhere?

Reply:
Yea, it makes sense. I will refactor all those messages in a separate pr.